contracts/QuestFactory.sol

```diff
@@ -31,6 +35,8 @@ contract QuestFactory is Initializable, OwnableUpgradeable {
 
     function initialize(address claimSignerAddress_, address rabbitholeReceiptContract_) public initializer {
         __Ownable_init();
+        __AccessControl_init();
+        grantDefaultAdminAndCreateQuestRole();
```

Comment:
Since we already deployed, I imagine that the `initialize` function is already called. Would we need to redeploy a new set of contracts?

Reply:
we don't need to, we just need to call `grantDefaultAdminAndCreateQuestRole` after deployment.